definitions/region/model_native_regions/image_v3.3.yaml

```
@@ -0,0 +1,28 @@
+# The model documentation and regional aggregation for IMAGE (TIMER) version 3.2 can be found at https://models.pbl.nl/image/index.php/Region_classification_map
+- IMAGE 3.2:
```

Also in the lines below...
```suggestion
- IMAGE 3.3:
```